apps/search-poc/src/views/DetailsContainer.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import useEntity from '../hooks/useEntity';
+import Details from '../components/Details';
+
+const DetailsContainer: React.FunctionComponent<{
+  params: {
+    selfUrl: string;
+  };
+}> = props => {
+  const { entity, loading, error } = useEntity(props.params.selfUrl);
```

Comment:
Not sure this hook is necessary... you can directly do:
```suggestion
  const { data, loading, error } = useNexus(nexus => nexus.httpGet({ path: props.params.selfUrl }))
```